spring-cloud-sleuth-brave/src/main/java/org/springframework/cloud/sleuth/brave/instrument/web/client/TraceWebClientAutoConfiguration.java

```diff
@@ -45,7 +43,6 @@
 @Configuration(proxyBeanMethods = false)
 @SleuthWebClientEnabled
 @ConditionalOnBean(HttpTracing.class)
-@AutoConfigureAfter(TraceHttpAutoConfiguration.class)
 @AutoConfigureBefore(HttpClientConfiguration.class)
```

Comment:
An auto-configuration outside of the dedicated `autoconfigure` package makes no sense to me.